recipes/pymccrgb/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+python setup.py --quiet install --single-version-externally-managed --record=record.txt
+pip install git+https://github.com/stgl/pymcc 
```

Comment:
Why add this here?

Reply:
This is a dependency of the pymccrgb package not on pypi. It was originally installed from a git repo as part of the environment.

Reply:
Is it required to upload pymcc to pypi, add a conda recipe, and install it via conda-forge channel?

Reply:
Yes.